### ISSUE TITLE
Implement Chatbot Feature in MERN Stack Tech Blog Web App

### ISSUE THREAD

**Issue by AnanyaJain25** [singhJasvinder101/MyBlog#63]
##Description:
We need to implement a chatbot feature in our MERN stack tech blog web app. The chatbot should have the following functionality:

Greet the user and ask how it can help them.
Display a list of blog genres/categories the user can choose from.
Allow the user to select a category and display relevant blogs.

**Comment by singhJasvinder101** [comment]
arr you working on the task ??

**Comment by AnanyaJain25** [comment]
yes actually I faced a lot of errors while merging my last push about password constraints so i have desicded to start everything again..will be working over all three tasks this weekend(password,forget password and chat(this one might take time) )but you will get some update this weekend . sorry for the delay

**Comment by sharmi4590** [comment]
@singhJasvinder101 Could you please assign me this task if it still stale.
